features/developer_portal/cms_toolbar.feature

```diff
@@ -14,6 +14,10 @@ Feature: CMS Toolbar
     And go to the homepage
     Then there should not be a CMS toolbar
 
+  #TODO: We have to change the Factory in order to properly setup the database
+  #      so we have a Provider with a Buyer with an admin user with
+  #      username: "john" and password: "123456"
+  @wip
```

Comment:
I think this scenario is no longer valid since https://github.com/3scale/porta/pull/3809